setup.py

```diff
@@ -80,6 +80,7 @@
         'vharfbuzz',
         'collidoscope',
         'stringbrewer'
+        'rich>=3.7.4'
```

Comment:
we do not specify version numbers on setup.py unless there is a known bug (or new feature) that makes the code only run with at least a certain minimum version. In those cases, we add a comment explicitely explaining the reasoning behind the minimum version required

new dependencies should be listed on `requirements.txt` as well, and there we specify the latest compatible version available. In the case of pich, it seems to be version 10.0.1 according to https://pypi.org/project/rich/

Reply:
Oh! Thanks! I totally miss that `requirements.txt`!